concourse/steps/images.py

```diff
@@ -18,12 +18,46 @@
 import product.model
 
 
-def image_reference_filter(include_regexes=(), exclude_regexes=()):
-    def image_to_str(image_reference: product.model.ContainerImage):
-        return image_reference.image_reference()
-
+def _create_filter(
+    include_regexes,
+    exclude_regexes,
+    value_transformation_function,
+):
     return reutil.re_filter(
         include_regexes=include_regexes,
         exclude_regexes=exclude_regexes,
-        value_transformation=image_to_str,
+        value_transformation=value_transformation_function,
+    )
+
+
+def image_reference_filter(include_regexes=(), exclude_regexes=()):
```

Comment:
short-cut with e.g. `lambda x: True` if neither include- not exclude-REs are given